README.md

```diff
@@ -49,6 +49,37 @@ rest = xpublish.Rest(
 ```
 
 
+## OGC EDR Spec Compliance
+
+This package attempts to follow [the spec](https://docs.ogc.org/is/19-086r6/19-086r6.html) as closely as possible, adding functionality where the value is demonstrable. Below are the implementations and a listing of spec compliance:
```

Comment:
```suggestion
This package attempts to follow [the spec](https://docs.ogc.org/is/19-086r6/19-086r6.html) where reasonable, adding functionality where the value is demonstrable. Below are the implementations and a listing of spec compliance:
```

We'll want to add a note that instead of hosting data at `/collections/{dataset_id}/{query_type}` we're using `/datasets/{dataset_id}/edr/{query_type}` since that isn't as close as possible, but fits in Xpublish's default structure.